scripts/users-permissions.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+USER="admin:pass"
+HEADER="Content-Type: application/json"
+
+# Alias for curl 
+curl="./bin/curl -u ${USER}"
+
+# Create "Teacher" team
+${curl} -d "name=Teacher" grafana:3000/api/teams
```

Comment:
The teachers team identifier should be in the response of this request. Maybe store it to avoid an extra request?